### PR TITLE
Fix for kraken.com

### LINKED ISSUE #7057: [Broken Website] Poor contrast on Kraken.com
# Broken Website Report

## Website Address
kraken.com

## Steps To Reproduce

- Go on kraken.com
- Login to your account
- Notice the poor contrast between the different boxes inside the page

## Expected Behavior

- The page should have more contrast between content boxes.

## Actual Behavior

- The page has poor contrast between content boxes.

## Screenshots

Screenshot_20211011_170010

## System Information

- OS: Linux
- Browser: Firefox 93
- Dark Reader Version: 4.9.39

## Additional Context
The contrast on the text seems to be fine though, only the background colors seem poor.

### PR DESCRIPTION
**Pull request by elyse0**
Fixes #7057

- Fixes active tab indicator in navbar

image

- Fixes 'price gauge'

image

- Fixes contrast in search box

image

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7557,6 +7557,23 @@ INVERT
 .serviceLogoImg
 .serviceIcon
 
+===============================
+
+kraken.com
+
+CSS
+header > nav > ul > li > a::after, ._3ku3YkWZbOVnguKutXHfK- {
+    background-color: var(--darkreader-neutral-text) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7566]: `._3ku3YkWZbOVnguKutXHfK-` what is that selector used for?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -7557,6 +7557,23 @@ INVERT
...
+}
+a.has-text-primary, div.has-text-primary, .kraken-cta, label,
+a.nav-create, a.nav-create-account, a.nav-login {
+    color: var(--darkreader-neutral-text) !important;
+    background-color: var(--darkreader-neutral-background) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7572]: I'm not able to find any of these selectors, are they on the account dashboard of some sort?

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (elyse0) [reply src/config/dynamic-theme-fixes.config:7566]: It's the 'price gauge', I don't see other way to get it

image

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (elyse0) [reply src/config/dynamic-theme-fixes.config:7572]: Yeah, they are in a couple of different places

- https://support.kraken.com/hc/en-us
**a.nav-create, a.nav-create-account, a.nav-login**

This is without the fix

image

This is with the fix

image

- https://www.kraken.com/en-us/institutions
**.kraken-cta** 

This is without the fix

image

With the fix

image

- https://www.kraken.com/prices?quote=USD
**label**

This is without the fix (there are also other labels)

image

This is with the fix

image

- https://futures.kraken.com/

a.has-text-primary, div.has-text-primary

This is without the fix
image

This is with the fix

image

Probably there are other fixes inside the account dashboard but I don't have an account so... I focused on the problems that **erwangse** pointed out in the issue 🐞

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7566]:
```css
span[data-testid^=asset-pair"][data-testid$="-low"] + div > :last-child
```

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7572]: >     * https://support.kraken.com/hc/en-us
>       **a.nav-create, a.nav-create-account, a.nav-login**

Ehm, this seems like a cosmetic change, as the page without Dark Reader also indicates that those 2 buttons don't have a background-color.



>     * https://www.kraken.com/en-us/institutions
>       **.kraken-cta**

ditto as above.



> a.has-text-primary, div.has-text-primary

ditto as above.

`label` is also questionable for me, but not as questionable as the other ones.

> Probably there are other fixes inside the account dashboard but I don't have an account so... I focused on the problems that **erwangse** pointed out in the issue lady_beetle

Only a few issues that **erwangse** pointed out seems relevant to be fixed as others are just _meh_.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (elyse0) [reply src/config/dynamic-theme-fixes.config:7566]: 😮  great, thanks!

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (elyse0) [reply src/config/dynamic-theme-fixes.config:7572]: Yeah, it's sometimes hard to read purple text on black background in that page

Okay, I'll make these changes!

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -7557,6 +7557,15 @@ INVERT
 .serviceLogoImg
 .serviceIcon
 
+===============================
+
+kraken.com
+
+CSS
+header > nav > ul > li > a::after, span[data-testid^="asset-pair"][data-testid$="-low"] + div > :last-child {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7565]:
```suggestion
header > nav > ul > li > a::after,
span[data-testid^="asset-pair"][data-testid$="-low"] + div > :last-child {
```

**Review by Gusted — COMMENTED** [review]
Just a little change. Then it's good to go.

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (elyse0) [reply src/config/dynamic-theme-fixes.config]: Done! (:

**Review by Gusted — APPROVED** [review]

**Comment by Gusted** [comment]
Thanks!

**Comment by elyse0** [comment]
> Thanks!

The previous run failed because I missed a "=". I've fixed it and run the tests

**Pull request merged**